libcrux-psq/src/protocol/session.rs

```diff
@@ -0,0 +1,57 @@
+use super::{
+    ecdh::PublicKey,
+    keys::{derive_session_key, AEADKey},
+    responder::ResponderRegistrationPayload,
+    transcript::Transcript,
+    TransportMessage,
+};
+
+/// The length of a session ID in bytes.
+pub const SESSION_ID_LENGTH: usize = 32;
+
+/// The length of a sessin key in bytes.
+pub const SESSION_KEY_LENGTH: usize = 32;
+
+pub struct SessionKey {
+    pub(crate) identifier: [u8; SESSION_ID_LENGTH],
+    pub(crate) key: AEADKey,
+}
+
+pub struct SessionState<'keys> {
```

Comment:
How is this session state being loaded/stored? And how are the referenced keys being filled in when this happens.